releasenotes/notes/support-annotation-on-snapshotter-function-ff9b7ba2daf10427.yaml

```diff
@@ -0,0 +1,2 @@
+---
+features: Support custom annotations for VolumeSnapshot and VolumeSnapshotContent resource when it gets created.
```

Comment:
do we really need release note for this? I remember @hairyhum asked for it, but just wanted to confirm because this release note doesn't seem to be useful to the users.

Reply:
I understand that this change only affects internal golang API.
I think it still changes the API, so makes sense to mention even if it's not exposed to the function args (maybe it should be).

Reply:
ok.